integration/setup/setup.go

```diff
@@ -69,36 +73,76 @@ func Setup(c Config) error {
 }
 
 // Resources install required charts.
-func Resources(c Config) error {
+func Resources(config Config) error {
 	var err error
 
 	{
-		err = c.Host.InstallBranchOperator("azure-operator", "azureconfig", template.AzureOperatorChartValues)
+		c := chartvalues.AzureOperatorConfig{
+			Provider: chartvalues.AzureOperatorConfigProvider{
+				Azure: chartvalues.AzureOperatorConfigProviderAzure{
+					Location: env.AzureLocation(),
+				},
+			},
+			Secret: chartvalues.AzureOperatorConfigSecret{
+				AzureOperator: chartvalues.AzureOperatorConfigSecretAzureOperator{
+					SecretYaml: chartvalues.AzureOperatorConfigSecretAzureOperatorSecretYaml{
+						Service: chartvalues.AzureOperatorConfigSecretAzureOperatorSecretYamlService{
+							Azure: chartvalues.AzureOperatorConfigSecretAzureOperatorSecretYamlServiceAzure{
+								ClientID:      env.AzureClientID(),
+								ClientSecret:  env.AzureClientSecret(),
+								SubsciptionID: env.AzureSubscriptionID(),
+								TenantID:      env.AzureTenantID(),
+								Template: chartvalues.AzureOperatorConfigSecretAzureOperatorSecretYamlServiceAzureTemplate{
+									URI: chartvalues.AzureOperatorConfigSecretAzureOperatorSecretYamlServiceAzureTemplateURI{
+										Version: env.CircleSHA(),
+									},
+								},
+							},
+						},
+					},
+				},
+				Registry: chartvalues.AzureOperatorConfigSecretRegistry{
+					PullSecret: chartvalues.AzureOperatorConfigSecretRegistryPullSecret{
+						DockerConfigJSON: env.RegistryPullSecret(),
+					},
+				},
+			},
+		}
+
+		values, err := chartvalues.NewAzureOperator(c)
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		err = c.Host.InstallStableOperator("cert-operator", "certconfig", e2etemplates.CertOperatorChartValues)
+
+		err = config.Release.InstallOperator(context.Background(), "azure-operator", release.NewVersion(env.CircleSHA()), values, providerv1alpha1.NewAzureConfigCRD())
```

Comment:
Let's please pass the context as an argument from top-level `Setup` call <3

Reply:
I will do this later. One step at a time. I do not yet have a picture where it should come from but we will get to this point. No worries.